shark_turbine/kernel/wave/iree_utils.py

```diff
@@ -23,6 +24,23 @@ def get_mmt_asm(lhs_type: str, rhs_type: str, acc_type: str) -> str:
     return matmul_function
 
 
+def get_conv_asm(
+    conv_type: str, lhs_type: str, rhs_type: str, res_type: str, stride: int
+) -> str:
+    res_dtype = res_type.split("x")[-1]
+    return f"""
+    func.func @conv_{conv_type}(%lhs: tensor<{lhs_type}>, %rhs: tensor<{rhs_type}>) -> tensor<{res_type}> {{
```

Comment:
Not related to this patch, but one way to enforce that iree using the "best" values for lowering pipeline etc would be to get the transform dialect scripts from the sdxl-scripts repo.